source/configuration_manual/sieve/examples.rst

```diff
@@ -0,0 +1,401 @@
+.. _sieve_examples:
+
+=========================
+Pigeonhole Sieve examples
+=========================
```

Comment:
the several extensions linked from this page should probably link to the configuration page where it exists and not to the standard directly.

Reply:
that's already being done.